atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -560,6 +561,17 @@ def get_build(self, metadata):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if help_result is not None:
+            try:
+                cgit_url = self.workflow.source.uri.replace('/rpms/', '/cgit/rpms/')
```

Comment:
I think this can be solved by modifying this plugin, or `AddHelpPlugin??`, to take a new param that specifies the template to be used to construct the URL.